dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/StoredDataRequestMessageObject.kt

```diff
@@ -2,12 +2,12 @@ package org.dataland.datalandcommunitymanager.model.dataRequest
 
 /**
  * Contains info about a stored message object on Dataland.
- * @param contactList  a list of strings which include all contact (mail) details
+ * @param contacts  a list of strings which include all contact (mail) details
  * @param message a string of all messages which were created during the life cycle
  * @param lastModifiedDate the date when the data request has been modified the last time
  */
 data class StoredDataRequestMessageObject(
```

Comment:
Ist das eigentlich nicht eher so etwas wie ein message-history-Objekt?

Der Name hat mich anfangs etwas irritiert.

Und das "stored" fühlt sich irgendwie im Namen auch falsch an.  Das weckt den Eindruck dass diese Klasse eine gespeicherte Entität repräsentiert (quasi Api-Model einer Entität).

Dabei handelt es sich eher um ein Hilfsobjekt, das im Code benutzt wird, aber nicht in der api.

Reply:
Die Api sollte das doch returnen als teil von der StoredDataRequest. Ich schätze daher auch der name. Deshalb hab ich es auch erstmal so gelassen. Aber warum da Object drinsteht ist mir auch schleierhaft.

Reply:
Passt